### PR TITLE
eyes test for tutorial landing pages

### PR DESCRIPTION
**Pull request by tanyaparker**

### CODE REVIEW

**Comment by tanyaparker** [comment]
I'm trying to run the eyes test locally first, but opening this PR for early feedback.

File: dashboard/test/ui/features/tutorial-landing-pages.feature [hunk 1]
```diff
@@ -0,0 +1,18 @@
+@eyes
+Feature: Looking at tutorial landing pages with Applitools Eyes
+
+Scenario Outline: Viewing tutorial landing page
+  Given I am on "http://code.org/"
```
Comment (breville) [inline dashboard/test/ui/features/tutorial-landing-pages.feature]: I'm not sure we need this line.

**Comment by breville** [comment]
This is very close to [other_sites.feature](https://github.com/code-dot-org/code-dot-org/blob/9478f382afbc30dfae542a917aab4838b6f6a7c8/dashboard/test/ui/features/other_sites.feature) and perhaps they could just be combined.  But at any rate, this seems like a great piece of coverage!

**Comment by tanyaparker** [comment]
@breville good call! Moved them over to other_sites.feature instead. PTAL

**Pull request merged**